src/main/java/hudson/plugins/sonar/action/SonarProjectIconAction.java

```diff
@@ -44,7 +44,7 @@ public SonarProjectIconAction(SonarAnalysisAction buildInfo) {
 
   @Override
   public String getIconFileName() {
-    PluginWrapper wrapper = Jenkins.getInstance().getPluginManager()
+    PluginWrapper wrapper = Jenkins.getInstanceOrNull().getPluginManager()
```

Comment:
sounds like a NPE could happen here?

Reply:
> sounds like a NPE could happen here?

In theory yes, but just like the `getInstance()` but this hides the fact the a NPE could get thrown.
In any case, the NPE is only thrown if Jenkins in not running, like in test cases, so I think we can safely ignore this warning in QG. One of the code bugs in Jenkins if you ask me with using static wiring.